examples/servicenow-mcp/README.md

```diff
@@ -0,0 +1,23 @@
+This is a simple example of making an MCP tool call against the
+servicenow MCP server from
+https://github.com/echelon-ai-labs/servicenow-mcp in SSE mode. Note
+that this server does not support authorisation. It uses credentials
+set in its environment (as described in installation guide at that
+link) to access the servicenow instance.
```

Comment:
If I understand the content at that link correctly, the way to start the server is:

```
servicenow-mcp-sse --instance-url=https://your-instance.service-now.com --username=your-username --password=your-password
```

Is that right?  If so, can you add that to this README?  Also, if you can point someone to a link for where to get started getting a trial instance, username, and password I think that would help a lot too.